src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ReflectionXmlFormatReader.cs

```diff
@@ -152,15 +152,27 @@ private void ReflectionSetMemberValue(object obj, object memberValue, MemberInfo
                         Type genericCollectionType = Globals.TypeOfICollectionGeneric.MakeGenericType(itemType);
                         if (genericCollectionType.IsAssignableFrom(propertyType))
                         {
-                             addMethod = genericCollectionType.GetMethod("Add");
+                            addMethod = genericCollectionType.GetMethod("Add");
```

Comment:
As with other commit, what happens if someone derives from Dictionary, or implements IDictionary but has an extra Add method? Would this code still get called?